src/main/java/com/troubleshooting/model/ConjurLogModel.java

```diff
@@ -0,0 +1,49 @@
+/**
+ * Model object for Conjur log input
+ */
+package com.troubleshooting.model;
+
+public class ConjurLogModel implements ILogModel {
```

Comment:
Create a constructor and populate all fields

Reply:
why constructor instead of getters/setters?

Reply:
less lines?

Reply:
Because your setting this object once.